hedera-node/test-clients/src/main/java/com/hedera/services/bdd/junit/TestBase.java

```diff
@@ -97,38 +50,6 @@ public static List<HapiSpec> extractContextualizedSpecsFrom(
                 .toList();
     }
 
-    protected final DynamicTest hgcaaLogValidation(final String loc) {
```

Comment:
Not an obvious change for `"feat: test-clients changes for block streams"` so for the record: These deletions in this fail are dead code cleanup.

Reply:
yes these were all unused 🙂

Reply:
Yikes!  I did **not** mean "FAIL"!  Probably I meant "PR" but it's hard to see how I can blame this on auto-correct.  Must have just been a brain fart.